sycl/test-e2e/ESIMD/raw_send.cpp

```diff
@@ -0,0 +1,181 @@
+//==---------------- raw_send.cpp  - DPC++ ESIMD on-device test ------------==//
```

Comment:
This test is a copy-paste + templatization of int main() from existing test vadd_raw_send.cpp.

Can you please replace the content of vadd_raw_send.cpp with the content of this new test + keep the test-case for 'float' type as it was in vadd_raw_send.cpp?

Reply:
Or alternatively, add test<float> to this test and remove the old vadd_raw_send.cpp